inc/Engine/Optimization/RUCSS/ServiceProvider.php

```diff
@@ -51,7 +51,7 @@ public function register() {
 			->withArgument( $this->getContainer()->get( 'rucss_usedcss_table' ) );
 
 		$this->getContainer()->add( 'rucss_used_css_query', 'WP_Rocket\Engine\Optimization\RUCSS\Database\Query\UsedCSS' );
-		$this->getContainer()->add( 'rucss_api_client', 'WP_Rocket\Engine\Optimization\RUCSS\APIClient' );
+		$this->getContainer()->add( 'rucss_frontend_api_client', 'WP_Rocket\Engine\Optimization\RUCSS\Frontend\APIClient' );
 		$this->getContainer()->add( 'rucss_used_css_controller', 'WP_Rocket\Engine\Optimization\RUCSS\Controller\UsedCSS' )
 			->withArgument( $this->getContainer()->get( 'rucss_used_css_query' ) );
 
```

Comment:
Question @Tabrisrp , will these need to be changed at all to conform to the new container (I'm thinking of the new tagging, etc., that was needed in the other recent container update PRs?

Reply:
yes they will